tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -31,7 +31,51 @@ sub run {
         # set blank screen to be never for current session
         script_run("gsettings set org.gnome.desktop.session idle-delay 0");
         become_root;
+        if (check_var('HDDVERSION', '12')) {
+            # use latest yast2-registration version because is not officially available
+            assert_script_run 'wget --quiet ' . data_url('yast2-registration-3.1.129.18-1.noarch.rpm');
+            zypper_call '--no-gpg-checks in yast2-registration-3.1.129.18-1.noarch.rpm';
+            record_info 'workaround', 'until the package is available for SLE 12 GA';
```

Comment:
does this need to specified on the release notes or something? Doesn't the customer have the same problem?